bin/build_and_run_all_tests

```diff
@@ -52,6 +52,7 @@ cd "${TEST_DIR}"
 
 MAKEFLAGS="${MAKEFLAGS:--j}"
 EXCLUDED_PDI_TESTS=""
+CMAKE_FLAGS=""
```

Comment:
We want to enable users to pass flags using the env variables, we shouldn't erase them